README.md

```diff
@@ -2,6 +2,9 @@
 
 A web service for proxying requests to the Esri print service switching out locked down quad-words for wide open ones.
 
+[!IMPORTANT]
+Looking for information on how to upgrade to v2? Head to [this doc](docs/v2-migration.md).
```

Comment:
**suggestion**: if you are wanting the fancy highlighting [notice](https://github.com/orgs/community/discussions/16925) here the syntax is a little off. 

```suggestion
> [!IMPORTANT]
> Looking for information on how to upgrade to v2? Head to [this doc](docs/v2-migration.md).
```